native_client/deepspeech.cc

```diff
@@ -434,6 +437,25 @@ DS_FreeMetadata(Metadata* m)
   }
 }
 
+void
+DS_FreeResult(Result* r)
+{
+  if (r) {
+    for (int i = 0; i < r->num_transcriptions; ++i) {
+      Metadata* m = &r->transcriptions[i];
```

Comment:
nit: `&r->transcriptions[i]` could use some `(` `)` to make it more obvious.

Also, checking `m != NULL` / asserting just in case.